file/readfile.go

```diff
@@ -132,3 +140,26 @@ func configFilesInDir(dir string) ([]string, error) {
 	}
 	return res, nil
 }
+
+func processEnvs(content string) (string, error) {
```

Comment:
This function is in fact a gotemplate rendering mapper. It happens to have `env` as its defined function, but may include other ones in the future.

I recommend that we redefine the responsibility of this function from "fill in env vars" to "render the deck file template" and rename it accordingly.

Reply:
Done.